src/ios/Accessibility.ts

```diff
@@ -89,6 +89,11 @@ export class Accessibility extends NativeAccessibility {
             }
         }
     }
+
+    private compareRawAnnouncements(postedAnnouncement: string, payloadAnnouncement: string): boolean {
+        const trimSpacesCommas = new RegExp(/[\s,]/);
```

Comment:
nit: in this case better to use RegEx literal instead of constructor `new RegExp(/[\s,]/)` -> `/[\s,]/`

Reply:
@a-tarasyuk We are not using dynamic input here, so why?

Reply:
> we are not using dynamic input here

```typescript
new RegExp(/[\s,]/)
```

constructor here is useless because you pass to it regular expression `/[\s,]/` which can be used without constructor

```
const r = /[\s,]/g;
```

Does that make sense?

Reply:
@a-tarasyuk Ok, I've refactored it a little bit, but I still believe that more strict way makes more sense.

Reply:
I meant in my previous comment, the following changes: 

```typescript

private _compareRawAnnouncements(postedAnnouncement: string, payloadAnnouncement: string): boolean {
    const spacesCommasRegExp = /[\s,]/g;
    return payloadAnnouncement.replace(spacesCommasRegExp, '') === postedAnnouncement.replace(spacesCommasRegExp, '');
}
```

Reply:
@a-tarasyuk I got it - still believe this is not TS way to do things)

Reply:
It is not related to TS, it common JavaScript practice, in which TypeScript will be [_transpired_](http://www.typescriptlang.org/play/index.html#src=const%20spacesCommasRegExp%20%3D%20new%20RegExp('%2F%5B%5Cs%2C%5D%2F'%2C%20'g')%3B)  

[Creating a regular expressionSection](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Creating_a_regular_expression
)
> Regular expression literals provide compilation of the regular expression when the script is loaded. If the regular expression remains constant, using this can improve performance.

> Using the constructor function provides runtime compilation of the regular expression. Use the constructor function when you know the regular expression pattern will be changing, or you don't know the pattern and are getting it from another source, such as user input.

Reply:
fixed